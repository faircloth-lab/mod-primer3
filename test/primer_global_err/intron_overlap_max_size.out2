..\src\primer3_core.exe: PRIMER_POS_OVERLAP_TO_END_DIST > PRIMER_MAX_SIZE / 2
