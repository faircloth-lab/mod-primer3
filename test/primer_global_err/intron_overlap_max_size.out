SEQUENCE_ID=Medicago Lyk3 mRNA INTRON_OVERLAP error big
SEQUENCE_TEMPLATE=cttgattcaatgaatctcaaaaatggattactattgttcattctgtttctggattgtgtttttttcaaagttgaatccaaatgtgtaaaagggtgtgatgtagctttagcttcctactatattataccatcaattcaactcagaaatatatcaaactttatgcaatcaaagattgttcttaccaattcctttgatgttataatgagctacaatagagacgtagtattcgataaatctggtcttatttcctatactagaatcaacgttccgttcccatgtgaatgtattggaggtgaatttctaggacatgtgtttgaatatacaacaaaagaaggagacgattatgatttaattgcaaatacttattacgcaagtttgacaactgttgagttattgaaaaagttcaacagctatgatccaaatcatatacctgttaaggctaagattaatgtcactgtaatttgttcatgtgggaatagccagatttcaaaagattatggcttgtttgttacctatccactcaggtctgatgatactcttgcgaaaattgcgaccaaagctggtcttgatgaagggttgatacaaaatttcaatcaagatgccaatttcagcataggaagtgggatagtgttcattccaggaagagatcaaaatggacatttttttcctttgtattctagaacaggtattgctaagggttcagctgttggtatagctatggcaggaatatttggacttctattatttgttatctatatatatgccaaatacttccaaaagaaggaagaagagaaaactaaacttccacaaacttctagggcattttcaactcaagatgcctcaggtagtgcagaatatgaaacttcaggatccagtgggcatgctactggtagtgctgccggccttacaggcattatggtggcaaagtcgacagagtttacgtatcaagaattagccaaggcgacaaataatttcagcttggataataaaattggtcaaggtggatttggagctgtctattatgcagaacttagaggcgagaaaacagcaattaagaagatggatgtacaagcatcgtccgaatttctctgtgagttgaaggtcttaacacatgttcatcacttgaatctggtgcggttgattggatattgcgttgaagggtcacttttcctcgtatatgaacatattgacaatggaaacttgggtcaatatttacatggtataggtacagaaccattaccatggtctagtagagtgcagattgctctagattcagccagaggcctagaatacattcatgaacacactgtgcctgtttatatccatcgcgacgtaaaatcagcaaatatattgatagacaaaaatttgcgtggaaaggttgctgattttggcttgaccaaacttattgaagttggaaactcgacacttcacactcgtcttgtgggaacatttggatacatgccaccagaatatgctcaatatggcgatgtttctccaaaaatagatgtatatgcttttggcgttgttctttatgaacttattactgcaaagaatgctgtcctgaagacaggtgaatctgttgcagaatcaaagggtcttgtacaattgtttgaagaagcacttcatcgaatggatcctttagaaggtcttcgaaaattggtggatcctaggcttaaagaaaactatcccattgattctgttctcaagatggctcaacttgggagagcatgtacgagagacaatccgctactacgcccaagcatgagatctatagttgttgctcttatgacactttcatcaccaactgaagattgtgatgatgactcttcatatgaaaatcaatctctcataaatctgttgtcaactagatgaagattttgtgtgacaaattgaattgtgtttgttaaaacatgtagaaagcatacaacaaatggtttgtactttacttgtatatgaaatattgcagttggagagtttttacttttcttacctcaattatccatcttgaacattgttttgtatgtggcaagagttcaaacactggtgtactcattgaaaagttatggtgagaaaatcactgatcagatgat
PRIMER_TASK=pick_detection_primers
PRIMER_PICK_RIGHT_PRIMER=1
PRIMER_PICK_INTERNAL_OLIGO=0
PRIMER_PICK_LEFT_PRIMER=1
PRIMER_FIRST_BASE_INDEX=1
PRIMER_POS_OVERLAP_TO_END_DIST=14
PRIMER_MAX_SIZE=27
PRIMER_ERROR=PRIMER_POS_OVERLAP_TO_END_DIST > PRIMER_MAX_SIZE / 2
=
